packages/esm-outpatient-app/src/home.component.tsx

```diff
@@ -1,15 +1,79 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PatientQueueHeader from './patient-queue-header/patient-queue-header.component';
 import ClinicMetrics from './patient-queue-metrics/clinic-metrics.component';
+import styles from './root.scss';
+import PatientSearchLaunch from './patient-search-icon/patient-search-icon.component';
+import Add16 from '@carbon/icons-react/es/add/16';
+import { Tile, Button, Switch, ContentSwitcher, Grid, Row, Column } from 'carbon-components-react';
+import { useTranslation } from 'react-i18next';
 
 interface HomeProps {}
 
+enum RouteStateTypes {
+  ADD_PATIENT_LIST,
+  ADVANCED_PATIENT_LIST,
+}
+//equivalent to attributes
+interface AddPatientRouteState {
+  type: RouteStateTypes.ADD_PATIENT_LIST;
+}
+
+interface AdvancedPatientListState {
+  type: RouteStateTypes.ADVANCED_PATIENT_LIST;
+}
+
+type RouteState = AddPatientRouteState | AdvancedPatientListState;
+
 const Home: React.FC<HomeProps> = () => {
+  const [routeState, setRouteState] = useState<RouteState>({ type: RouteStateTypes.ADD_PATIENT_LIST });
```

Comment:
Why not just 

```suggestion
  const [routeState, setRouteState] = useState<RouteState>(RouteState.ADD_PATIENT_LIST);
```

and simplify the types accordingly?